docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -726,3 +726,73 @@ Tasks are not running, and the server logs contain the following error message:
+--------------------------------------------------
+
+This log message tells us that Task Manager is not managing to keep up with the sheer amount of work it has been tasked with completing. This might mean that Alerts are not running at the frequency that was expected (instead of running every 5 minutes, it runs every 7-8 minutes, just as an example).
+
+By default Task Manager is limited to 10 tasks and this can be bumped up by setting a higher number in the kibana.yml file using the xpack.task_manager.max_workers configuration. That said, though, it’s important to keep in mind that a higher number of tasks running at any given time means more load on both Kibana and Elasticsearch, so only do so if adding this additional load on the environment makes sense.
+
+Another approach to addressing this might be to tell workers to run at a higher rate, rather than adding more of them, which would be configured using xpack.task_manager.poll_interval. This value dictates how often Task Manager checks to see if there’s more work to be done and uses milliseconds (by default it is 3000, which means an interval of 3 seconds).
+
+Before changing either of these numbers it’s highly recommended to investigate what Task Manager can’t keep up - Are there an unusually high number of alerts in the system? Are alerts failing often, forcing Task Manager to re-run them constantly? Is Kibana under heavy load? There could be a variety of issues, none of which should be solved by simply changing these configurations.
+
+Task TaskType failed in attempt to run:
+[source, txt]
+--------------------------------------------------
+server log [12:41:33.672] [info][plugins][taskManager][taskManager] Task TaskType "alerting:example.always-firing" failed in attempt to run: Unable to load resource ‘/api/something’
+--------------------------------------------------
+
+This log message tells us that when Task Manager was running one of our alerts, it’s task errored and, as a result, failed. In this case we can tell that the alert that failed was of type alerting:example.always-firing and that the reason it failed was Unable to load resource ‘/api/something’ . This is a contrived example, but broadly, if you see a message with this kind of format, then this tells you a lot about where the problem might be.
+
+For example, in this case, we’d expect to see a corresponding log line from the Alerting framework itself, saying that the alert failed. You should look in the Kibana log for a line similar to the log line below (probably shortly before the Task Manager log line):
+
+Executing Alert "27559295-44e4-4983-aa1b-94fe043ab4f9" has resulted in Error: Unable to load resource ‘/api/something’
+
+This would confirm that the error did in fact happen in the Alert itself (rather than the Task Manager) and it would help us pin-point the specific ID of the alert which failed: 27559295-44e4-4983-aa1b-94fe043ab4f9
```

Comment:
```suggestion
This would confirm that the error did in fact happen in the rule itself (rather than the Task Manager) and it would help us pin-point the specific ID of the rule which failed: 27559295-44e4-4983-aa1b-94fe043ab4f9
```